src/index.css

```diff
@@ -1,27 +1,73 @@
-body {
+:root {
+  --screen-color: #858693;
+  --dark: #000;
+  --white: #fff;
+  --operator-color: #eb8537;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  border-radius: 0;
+}
+
+#root {
+  text-align: center;
+}
+
+.container {
+  background-color: #fff;
+  height: auto;
+  margin: 40px;
+  max-width: 480px;
+  padding: 10px;
+  width: 100%;
+}
+
+.rowBtn {
+  width: 400px;
+  display: grid;
+}
```

Comment:
- [ ] Next, let's give the `.rowBtn` a width of 100% too. Using fixed widths is what's creating the problem.